### PR TITLE
add prompts

### PR DESCRIPTION
**Pull request by michaelneale**
Some real world prompts in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,11 +32,37 @@ goose session start
 You will see a prompt `G❯`:
 
 ```
-G❯ type your instructions here exactly as you would tell a developer.
+G❯ Looking at the in progress changes in this repo, help me finish off the feature. CONTRIBUTING.md shows how to run the tests.
```
Comment (baxen) [inline README.md]: nit: maybe move this down to dedicated examples and leave the original instruction? i think its still helpful to make it clear that any general text goes in this prompt

File: README.md [hunk 2]
```diff
@@ -32,11 +32,37 @@ goose session start
...
+```
+
+> [!NOTE]
+> From here you can talk directly with goose - send along your instructions. If you are looking to exit, use `CTRL+D`,
+> although goose should help you figure that out if you forget. More examples:
```
Comment (baxen) [inline README.md]: nit: like above, i'd move all the examples inside the note i think (or alternatively all outside)

**Review by baxen — APPROVED** [review]
Nice!

File: README.md (hunk 2 above)
Comment (michaelneale) [reply README.md]: yep, have moved it to an example section, makes sense

**Comment by FatStinkyPanda** [comment]
Claimed by scout - starting now.

**Comment by FatStinkyPanda** [comment]
Claimed by scout - starting now.

**Comment by FatStinkyPanda** [comment]
Claimed by scout - starting now.

**Comment by FatStinkyPanda** [comment]
Claimed by scout - starting now.

**Pull request merged**